dobi.yaml

```diff
@@ -0,0 +1,93 @@
+
+meta:
+    project: assembl
+
+
+mount=source:
+    bind: .
+    path: /opt/assembl
+
+mount=wheels:
+    bind: wheels/
+    path: /opt/wheels
+
+image=faber:
+    image: assembl-faber
+    context: docker/
+    dockerfile: Dockerfile.fabric
+
+#
+# Create a database image with the correct user and fixtures loaded
+#
+job=setup-postgres:
+    use: faber
+    mounts: [source, wheels]
+    net-mode: '{project}_default'
+    command: "fab devenv setup_postgresql_docker"
+    depends: [empty-database]
+
+compose=empty-database:
+    files: [docker/compose/empty-db.yaml]
+    project: '{project}'
+
+#
+# Build local wheels so that dev dependencies don't need to be installed in the 
+# runtime image.
+#
+image=wheeler:
+    image: assembl-wheeler
+    context: docker/
+    dockerfile: Dockerfile.wheel
+
+job=build-wheels:
```

Comment:
I do not have yet enough knowledge about wheels to understand the whole wheels part of the PR (and what it improves compared to the current status).